further improved the observability strategy document (#713)
kyma-project/community: concepts/observability-strategy/configurable-tracing/01-tracing-groundwork.md

- The collector must run stably at any time when using the typical settings. Bad configuration must be prevalidated and rejected. Fast feedback is welcome.
- Outputs
  - Support configuration of backends and outputs at runtime (no need to run a Kyma upgrade process) in a scenario-focused approach.

>   - As a minimum, support a single configuration with the potential to support multiple configuration at the same time in future (in individual Kubernetes resources).

Reviewer: ```suggestion
  - As a minimum, support a single configuration with the potential to support multiple configuration at the same time in the future (in individual Kubernetes resources).
```

  - As a minimum, support one vendor-neutral output (OTLP). It should be possible to chain your custom collector for specific conversions. For example, supporting the OLTP protocol will support most of the vendors already. Chaining a custom OpenTelemetry Collector can do custom conversion to a specific protocol.
  - Support typical settings for the supported outputs in a convenient way.
  - Non-typical settings might be possible as well (not mandatory), leaving the collector in an unsupported mode.
  - Secrets must be kept secret.
  - Secret bindings for the SAP BTP Operator must be supported, including secret rotations.

>   - WHen having multiple scenarios active at the same time (not part of minimal setup), then Scenarios must be isolated and have their own buffer management. If one backend is in a bad shape and cannot process any data anymore, data should still continue to be pushed to other backends.

Reviewer: ```suggestion
  - When multiple scenarios are active at the same time (not part of minimal setup), the scenarios must be isolated and have their own buffer management. If one backend is in a bad shape and cannot process data anymore, data should still continue to be pushed to other backends.
```

>   - Typical auth mechanisms for the integration must be supportable, minimum is basic auth but preferable client-certificate-based solutions.

Reviewer: ```suggestion
  - Typical auth mechanisms for the integration must be supportable, minimum is basic auth but preferable are client-certificate-based solutions.
```

- Inputs
  - Inputs are predefined. In best case, users do not need to differentiate between inputs.
- Filter

>   - Filtering of data (like dropping attributes or whole traces) will be needed. However, minimal settings are influencing the tail-based sampling in a static way (percentage)

Reviewer: ```suggestion
  - Filtering of data (like dropping attributes or whole traces) is needed. However, minimal settings influence the tail-based sampling in a static way (percentage).
```
